elf_reader_test.go

```diff
@@ -329,7 +329,7 @@ func TestGetProgType(t *testing.T) {
 	}
 
 	for _, tc := range testcases {
-		pt, at, to := getProgType(tc.section)
+		pt, at, _, to := getProgType(tc.section)
 		if pt != tc.pt {
```

Comment:
You can change this code to use `cmp.Diff` like elsewhere in the tests if you want.

Reply:
done